automated-comments/go/raindrops/string_builder_used.md

```diff
@@ -0,0 +1,3 @@
+The `%{function}` is helpful when concatenating a lot of little parts to a string.
+In a use case like this it is overkill. Here the `+=` operator is faster and uses less memory.
```

Comment:
I'm not entirely sure about the word "overkill". It sounds a little bit, well, dramatic. Maybe something like:

```suggestion
For this use case, the `+=` operator is a bit simpler, faster and uses less memory.
```